recipes/fastscapelib-f2py/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "fastscapelib-f2py" %}
+{% set version = "2.7.0" %}
+{% set sha256 = "20a44c8ef82049a89df6429e641a4ca5fbe04a2434e161483a397a43524f8f66" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/fastscape-lem/fastscapelib-fortran/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # problem with python2.7 configuration on windows and osx (from slycot-feedstock)
+  skip: true                     # [win]
+  skip: true                     # [osx and py==27]
+
+# TODO: (see slycot-feedstock): issues with clang compiler on osx -> use gcc
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}  # [unix]
+    - flang                      # [win]
+    - {{ compiler('c') }}        # [not osx]
```

Comment:
Can you leave a comment why this doesn't work for osx?

Reply:
I've added more details in the TODO comment that was just before the `requirements` section.

Reply:
Can you send a log with the error you get? You might not have setup CONDA_BUILD_SYSROOT correctly.

Reply:
Here is a link to the [log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=36594&view=logs&jobId=c572e5d5-7ba7-55ce-5575-7ce1d9f7161e&taskId=81967b5f-0283-5c0b-87c1-e8fdeef842a3&lineStart=238&lineEnd=239&colStart=1&colEnd=1) of the azure pipeline run for osx (error at line 238).